### PR TITLE
1069 show message when no metadata is present

### PR DESCRIPTION
**Pull request by tcnichol**
I added some messages here if there is no extractor or user metadata, for datasets or files. 

@lmarini I used the text you had supplied in the issue for the user metadata, and came up with something else for machine metadata. I think the text in those spaces can be better, but I wanted this to be a draft now so others can weigh in on the messages.

### CODE REVIEW

File: frontend/src/components/metadata/DisplayMetadata.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
 import { Agent } from "./Agent";
 import { MetadataDeleteButton } from "./widgets/MetadataDeleteButton";
 import { fetchPublicDatasetMetadata } from "../../actions/public_dataset";
+import { Link as RouterLink } from "react-router-dom";
```
Comment (longshuicy) [inline frontend/src/components/metadata/DisplayMetadata.tsx:16]: I didn't find where the RouterLink is being used. Could you remove that?

File: frontend/src/components/metadata/DisplayMetadata.tsx [hunk 2]
```diff
@@ -117,61 +118,76 @@ export const DisplayMetadata = (props: MetadataType) => {
...
-						}
+									</Box>
+								);
+							}
+						});
 					});
-				});
+				} else {
+					return (
+						<Box textAlign="center">
```
Comment (longshuicy) [inline frontend/src/components/metadata/DisplayMetadata.tsx:179]: The center looks a bit weird. Could it be just align on the left? 
Thoughts? 
image

**Review by longshuicy — CHANGES_REQUESTED** [review]
Coud you resolve merge conflict?

**Review by ddey2 — CHANGES_REQUESTED** [review]
It always says 'no user metadata provided for dataset', even for files.
Screenshot 2024-06-28 at 12 22 39 PM

File: frontend/src/components/metadata/DisplayMetadata.tsx (hunk 2 above)
Comment (tcnichol) [reply frontend/src/components/metadata/DisplayMetadata.tsx]: Aligned left.

**Comment by tcnichol** [comment]
> It always says 'no user metadata provided for dataset', even for files. Screenshot 2024-06-28 at 12 22 39 PM

I changed 'dataset' to 'resource' so that both messages just say 'resource'.

File: frontend/src/components/metadata/DisplayMetadata.tsx (hunk 1 above)
Comment (tcnichol) [reply frontend/src/components/metadata/DisplayMetadata.tsx:16]: removed.

**Review by longshuicy — APPROVED** [review]
Looks good.

**Pull request merged**